### ISSUE TITLE
SphereFace: Deep Hypersphere Embedding for Face Recognition

### ISSUE THREAD

**Issue by joyhuang9473** [number9473/nn-algorithm#83]
# SphereFace: Deep Hypersphere Embedding for Face Recognition #

- Author: Weiyang Liu, Yandong Wen, Zhiding Yu, Ming Li, Bhiksha Raj, Le Song
- Origin: https://arxiv.org/abs/1704.08063
- Related:
  - https://github.com/wy1iu/sphereface
  - https://github.com/clcarwin/sphereface_pytorch
  - https://github.com/pppoe/tensorflow-sphereface-asoftmax
  - https://github.com/hujun100/tensorflow-sphereface
  - A new loss function based on sphereface & triplet loss https://github.com/KaleidoZhouYN/Angular-Triplet-Loss

**Comment by joyhuang9473** [comment]
#38

**Comment by joyhuang9473** [comment]
https://github.com/clcarwin/sphereface_pytorch

**Comment by joyhuang9473** [comment]
KaleidoZhouYN/Loss-Functions: My own understanding on various kinds of Loss Function

https://github.com/KaleidoZhouYN/Loss-Functions

**Comment by joyhuang9473** [comment]
https://github.com/pppoe/tensorflow-sphereface-asoftmax

**Comment by joyhuang9473** [comment]
https://github.com/hujun100/tensorflow-sphereface

**Comment by joyhuang9473** [comment]
https://github.com/KaleidoZhouYN/Angular-Triplet-Loss
